test/gui/tst_addAccount/test.feature

```diff
@@ -5,26 +5,45 @@ Feature: adding accounts
     So that I can sync data with various organisations
 
 
-	Background:
+    Background:
         Given user "Alice" has been created on the server with default attributes and without skeleton files
 
     Scenario: Adding normal Account
         Given the user has started the client
         When the user adds the first account with
-          | server      | %local_server%           |
-          | user        | Alice                    |
-          | password    | 1234                     |
-          | localfolder | %client_sync_path_user1% |
+            | server      | %local_server%           |
+            | user        | Alice                    |
+            | password    | 1234                     |
+            | localfolder | %client_sync_path_user1% |
         Then an account should be displayed with the displayname Alice Hansen and host %local_server_hostname%
 
 
-     Scenario: Adding multiple accounts
+    Scenario: Adding multiple accounts
         Given user "Brian" has been created on the server with default attributes and without skeleton files
         And user "Alice" has set up a client with default settings
         When the user adds an account with
-          | server      | %local_server%           |
-          | user        | Brian                    |
-          | password    | AaBb2Cc3Dd4              |
-          | localfolder | %client_sync_path_user2% |
+            | server      | %local_server%           |
+            | user        | Brian                    |
+            | password    | AaBb2Cc3Dd4              |
+            | localfolder | %client_sync_path_user2% |
         Then an account should be displayed with the displayname Alice Hansen and host %local_server_hostname%
         And an account should be displayed with the displayname Brian Murphy and host %local_server_hostname%
+
+
+    Scenario: Adding account with wrong credentials
+        Given the user has started the client
+        When the user adds the account with wrong credentials
+            | server   | %local_server% |
+            | user     | Alice          |
+            | password | 12345          |
+        Then error "The provided credentials are not correct" should be displayed
+
+
+    Scenario: Adding account with self signed certificate for the first time
+        Given the user has started the client
+        When the user adds an account with the following secure server address
```

Comment:
we can reuse it for unsecure server too.
```suggestion
        When the user adds an account with the following server address
```

Reply:
For unsecure server we generally enter other values like username and password along with the url and then click on next. But this step just enters the url so I think it is better this way.

Reply:
Agreed!! But we might need a step only to add server url (no matter secure or unsecure) and then click next like here.
So if you have a step like `the user adds the following server address` (enter url and hit next) then this can be reused.

Reply:
Yes we can do that but I think it is not needed here in this PR maybe we can make an issue to refactor that